cmd/tailscale/ui.go

```diff
@@ -1277,14 +1278,21 @@ func (ui *UI) showMessage(gtx layout.Context, msg string) {
 	op.InvalidateOp{}.Add(gtx.Ops)
 }
 
+func MaxDp(p1, p2 unit.Dp) unit.Dp {
```

Comment:
Just max and use generics? ;)

(I don't actually care)

Reply:
So far as I can see Gio has moved to integer coordinates everywhere and no longer uses float in its APIs. It just uses float internally now. I don't expect to need a Max for anything other than Dp (Display Pixels), so I'll plan to submit this as it is after a bit more testing.